docs/content/develop/templates.md

```diff
@@ -0,0 +1,280 @@
+
+    ```
+    // my-component.html:
+	<div class="custom">
+	    <slot name="header"></slot>
+	    <slot></slot>
+	    <slot name="footer"></slot>
+	</div>
+    ```
+
+
+*   If you put some content between the opening & closing tags of a web component without a slot name, it will put that markup into the unnamed slot.
+    *   If there is no unnamed slot, it will _not_ render that content. 
+    *   For this reason, it’s a good idea to leave one unnamed slot if general markup is allowed within that component. All PatternFly Element web components operate this way. 
+*   Whenever you add slot="something", you are telling the webcomponent where to put this information within the inner template.
+
+    ```
+    // my-web-page.html:
+	<pfe-card>
+	      <div slot="pfe-card--header">
+	          <h1>This heading text will appear in the card header region</h1>
+	      </div>
+	</pfe-card>
+    ```
+
+
+*   You can’t have nested slots _with the same name_, but when you use different names, the web component will pick up the content and put it where it belongs in the web component template, regardless of order in the light DOM.
+
+	```
+	// my-web-page.html:
+	<pfe-component>
+	  <div slot="header">
+	    <h1 slot="header">Nope, this is no good</h1>
+	  </div>
+	</pfe-component>
+	
+	// But...
+	<pfe-component>
+	  <div slot="header">
+	    <h1 slot="header__content">Yep, this will work</h1>
+	  </div>
+	</pfe-component>
+	
+	```
+
+
+
+*   The direct descendant inside a slot in the light DOM will accept styles from the web component. For example, we are able to style links within the CTA component because the `<a>` tag is being passed into the unnamed slot in the CTA component. The link tag doesn’t need an explicit attribute like` slot="link"` because if a web component has one unnamed `<slot></slot>`, then anything you put inside that custom component tag will be in that slot by default. 
+
+    ```
+    <pfe-cta priority="primary">
+      <a href="#">Primary</a>   <!-- this element is in the default slot -->
+    </pfe-cta>
+    ```
+
+
+*   Child elements within a custom tag don’t have to be the first child to be styled, they only have to be direct descendants of the component. Meaning once you nest something inside another tag, it can no longer receive styles targeted with the `::slotted` pseudo selector. Assuming the component has some basic styles on all slots like this:  `::slotted()  {border: red solid 1px;}` then both the div and H2 tag would recieve a red border:
+
+    ```
+    <pfe-cta priority="primary">
+      <div>styled!</div>
+      <h2 href="#">styled!</h2>
+    </pfe-cta>
+    ```
+
+    However, the nested `<a>` tag would not receive styles, because it’s not a direct child of the pfe-cta component.
+
+
+    ```
+    <rh-cta priority="primary">
+      <div>
+        <a href="#">cannot receive styles, because it's nested</a>
+      </div>
+    </rh-cta>
+
+    ```
+
+
+
+
+# Dynamic HTML Templates
+
```

Comment:
Maybe a @TODO note here?

Reply:
not sure what the todo pertains to?

Reply:
I was confused - thinking it was a heading with no content.  Nevermind!